collection-scripts/gather_vms_details

```diff
@@ -19,8 +19,29 @@ function gather_vm_info() {
   /usr/bin/oc adm inspect --dest-dir "${BASE_COLLECTION_PATH}" -n "${ocproject}" pod "$ocvm"
   /usr/bin/oc adm inspect --dest-dir "${BASE_COLLECTION_PATH}" -n "${ocproject}" virtualmachineinstances "${vmname}"
 
+  # VM : capabilities
+  /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh -r capabilities  > "${vm_collection_path}/${ocvm}.capabilities.xml"
+
+  # VM : domcapabilities
+  /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh  domcapabilities  > "${vm_collection_path}/${ocvm}.domcapabilities.xml"
+
+  # VM : list
+  /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh -r list --all  > "${vm_collection_path}/${ocvm}.list.txt"
+
   # VM : dumpxml
-  /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh dumpxml "${ocproject}_${vmname}" > "${vm_collection_path}/${ocvm}.dumpxml.xml"
+  /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh -r dumpxml "${ocproject}_${vmname}" > "${vm_collection_path}/${ocvm}.dumpxml.xml"
+
+  # VM : domblklist
+  /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh -r domblklist "${ocproject}_${vmname}" > "${vm_collection_path}/${ocvm}.domblklist.txt"
+
+  # VM : domjobinfo
+  /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh -r domjobinfo "${ocproject}_${vmname}" > "${vm_collection_path}/${ocvm}.domjobinfo.txt"
+
+   # VM : blockjob
+#   for disk in $(cat "${vm_collection_path}/${ocvm}.domblklist.txt" | awk '{print $2}'|grep -v Source):
+#    do
+#      /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh -r blockjob "${ocproject}_${vmname}" $disk > "${vm_collection_path}/${ocvm}.blockjob.txt"
+#    done
```

Comment:
@tiraboschi Hi, this part creates an empty `.blockjob.test` file, although the loop seems to be working. Any idea why?